### PR TITLE
fix: prevent flex

### PR DESCRIPTION
**Pull request by jouni**
Prevent Avatar from shrinking or growing inside a flex container. It can potentially stark shrinking if there is not enough space inside a flex container. But Avatar should always remain fixed size and preserve 1:1 aspect ratio.

### CODE REVIEW

**Comment by jouni** [comment]
Just realized, that once this is merged, this CSS in Avatar Group will be redundant, and should be removed: https://github.com/vaadin/vaadin-avatar/blob/master/src/vaadin-avatar-group.html#L42-L44

Should add that change to this PR…

File: src/vaadin-avatar.html [hunk 1]
```diff
@@ -14,6 +14,7 @@
     <style>
       :host {
         display: inline-block;
+        flex: none;
```
Comment (lkraav) [inline src/vaadin-avatar.html:17]: Hm, does this `display: inline-block` rule have any meaning as a flex child - could it be removed?

Comment (jouni) [reply src/vaadin-avatar.html:17]: Not sure if we could remove it when Avatar is used inside Avatar Group. But Avatar can be used standalone as well, and then it needs to be able to define a size for itself – `inline` won’t work.

**Comment by web-padawan** [comment]
Rebased and updated to remove styles mentioned at https://github.com/vaadin/vaadin-avatar/pull/95#issuecomment-740160894

**Review by web-padawan — APPROVED** [review]

**Pull request merged**